src/redfishpower/redfishpower.c

```diff
@@ -1589,9 +1589,10 @@ static void shell(CURLM *mh)
                         err_exit(false, "private data not set in easy handle");
 
                     if (cmsg->data.result != 0) {
-                        if (cmsg->data.result == CURLE_COULDNT_CONNECT
-                            || cmsg->data.result == CURLE_OPERATION_TIMEDOUT)
-                            printf("%s: %s\n", pm->plugname, "network error");
+                        if (cmsg->data.result == CURLE_COULDNT_CONNECT)
+                            printf("%s: %s\n", pm->plugname, "couldn't connect");
+                        else if (cmsg->data.result == CURLE_OPERATION_TIMEDOUT)
+                            printf("%s: %s\n", pm->plugname, "operation timedout");
```

Comment:
Would this be useful instead?

https://curl.se/libcurl/c/curl_multi_strerror.html

Reply:
Ahhh good idea, although b/c this is CURLcode not CURLMcode

https://curl.se/libcurl/c/curl_multi_info_read.html

we gotta use a slightly different function.